src/domain/event/enrol/SuccessToast.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FunctionComponent, useState, useEffect } from 'react';
+import { CSSTransition } from 'react-transition-group';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+
+import styles from './successToast.module.scss';
+import tada from '../../../assets/icons/svg/tada.svg';
+import Icon from '../../../common/components/icon/Icon';
+import { justEnrolled } from '../state/EventActions';
+import { justEnrolledSelector } from '../state/EventSelectors';
+
+const SuccessToast: FunctionComponent = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+
+  const [inProp, setInProp] = useState(false);
+  const isJustEnrolled = useSelector(justEnrolledSelector);
+
+  useEffect(() => {
+    setInProp(isJustEnrolled);
```

Comment:
We can't call a selector inside a `useEffect` call, because it is against the rules of hooks. If I put the useSelector call inside useEffect TypeScript throws an error: 

`React Hook "useSelector" cannot be called inside a callback. React Hooks must be called in a React function component or a custom React Hook function.eslint(react-hooks/rules-of-hooks)
`